packages/nuxi/src/commands/preview.ts

```diff
@@ -16,8 +17,11 @@ export default defineNuxtCommand({
   async invoke (args) {
     process.env.NODE_ENV = process.env.NODE_ENV || 'production'
     const rootDir = resolve(args._[0] || '.')
+    const { loadNuxtConfig } = await loadKit(rootDir)
+    const config = await loadNuxtConfig({ cwd: rootDir })
+    const serverOutputDir = config.nitro.output?.dir?.replace(/^~\//, '') || '.output'
```

Comment:
Probably not supporting `~` (it is srcDir while config is supposed to be from rootDir/workspaceDir)